src/routes/docs/tooling/appwriter/+page.markdoc

```diff
@@ -40,9 +40,49 @@ The keyboard comes with Mac layout keys pre-installed for macOS which can also b
 |Switch support|Hot-swappable switches with 3 & 5 pin|
 |Rollover|N-key rollover|
 
-# How to change the backlight {% #how-to-change-the-backlight %}
+# Quick start  {% #quick-start %}
 
-Press `Fn + Home` to change the RGB backlight effect and cycle through all the lights.
+There are currently three modes to connect the Appwriter to the computer: wired, wireless, and Bluetooth. There is also a switch under the keyboard to toggle between B, T, or G, which switch to wired, wireless, or Bluetooth mode, respectively.
+
+## Wired mode {% #wired-mode %}
+
+To use the wired mode, connect the USB-C data cable from the keyboard to the computer and push the Appwriter switch towards the T.
+When connecting the keyboard for the first time using wired mode, your computer will install all required drivers on a successful connection.
+
+The Appwriter's battery will charge while connected using the wired mode. Make sure to use a charger that doesn't exceed a voltage/current = DC5V = 1A (max). Using a charger that exceeds this voltage and current ratings could easily damage your keyboard.
+
+## Wireless mode {% #wireless-mode %}
+
+Push the Appwriter switch towards the G if you want to use the wireless mode with the wireless receiver. After switching to wireless mode, long press the `FN` + `4` keys for around 3 seconds. After the `4` key light flashes quickly, plug the receiver into a computer USB port.
```

Comment:
> long press the `FN` + `4` keys for around 3 seconds. After the `4` key light flashes quickly, plug the receiver into a computer USB port.

Instead of saying to press for 3 seconds, let's say -> long press the `FN` + `4` keys until the `4` key light flashes quickly